### PR TITLE
UI styling

### PR DESCRIPTION
**Pull request by coder1033**
## Description
I've used bootstrap and pug templating engine

There are total 4 views
- layout.pug
- index.pg
- form.pug
- error.pug

### layout.pug
this view contains the navbar which also contains the button to /new page.

### index.pug
This view is responsible for showing all the messages.

### form.pug
This view will show up on /new page. And it shows a form for adding a message.

### error.pug
This view will show up on all the routes other than `/` and `/new`

### CODE REVIEW

File: views/index.pug [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+      div(
+        class="card-size",
+        style={display:"inline-block"}
+        ).card.border-warning.m-3
+        div.card-header.text-center
+          h5= msg.user
+        div.text-right
+          small.text-muted= msg.added
+        div.card-body
+          p.card-text= msg.text
```
Comment (GaurisankarJ) [inline views/index.pug:17]: Add a new line please.

File: views/layout.pug [hunk 2]
```diff
@@ -3,6 +3,19 @@ html
...
+    script(src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js")
+
+    meta(name="viewport" content="width=device-width, initial-scale=1")
   body
-    h1 Welcome to Mini Message Board
+    nav(id="navbar").navbar.navbar-dark.bg-dark
+      div.container
+        a(href="http://localhost:3000/", id="headerlink").navbar-brand.display-1  Mini Message Board
+        a(href="http://localhost:3000/new").navbar-item.nav-link.navbar-right.btn.btn-outline-warning.btn-sm New
     block content
```
Comment (GaurisankarJ) [inline views/layout.pug:21]: Add a new line please.

File: views/index.pug (hunk 1 above)
Comment (coder1033) [reply views/index.pug:17]: Like I need to left a balnk line before starting a div. Right?

Comment (GaurisankarJ) [reply views/index.pug:17]: At the end at line 17, add a new line.

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: node ./bin/www
```
Comment (GaurisankarJ) [inline Procfile:1]: Add a new line.

File: public/stylesheets/style.css [hunk 4]
```diff
@@ -1,8 +1,21 @@
...
+  width: 16rem;
+}
+@media(max-width:500px){
+  #headerlink{
+    font-size: 22px;
+  }
+  .card-size{
+    width: 13rem;
+  }
+}
```
Comment (GaurisankarJ) [inline public/stylesheets/style.css:21]: Add a new line.

File: routes/index.js [hunk 5]
```diff
@@ -1,17 +1,24 @@
 var express = require("express");
```
Comment (GaurisankarJ) [inline routes/index.js:1]: Use const and not var for imports.

File: views/form.pug [hunk 6]
```diff
@@ -1,24 +1,28 @@
...
+        label(for="message") Message
+        textarea(
+          name="message",
+          placeholder="enter message",
+          autocomplete="off",
+          maxlength="200",
+          required
+        ).form-control
+      
+      button(type="submit").btn.btn-warning.btn-sm Add
```
Comment (GaurisankarJ) [inline views/form.pug:28]: Add a new line.

**Pull request merged**